CONTRIBUTING.md

```diff
@@ -59,6 +59,12 @@ Changes should conform to:
 
 With the exception that 4-space indendation be used.
 
+#### General guidelines for Swift for TensorFlow API docs contribution
```

Comment:
"Swift for TensorFlow" can be removed; it's implicit and isn't mentioned elsewhere in the document.

```suggestion
#### API documentation contribution guidelines
```

Reply:
Thanks @dan-zheng, you rock!

Reply:
Done